terraform/resources/aca/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
+      version = "3.0.2"
+    }
+  }
+}
+
+provider "docker" {
+  # Note: Terraform will automatically communicate with the local 
+  # Docker daemon using the default Unix socket. Change this only if your
+  # worker machine or deployment agent runs docker in another location. 
+  host = "unix:///var/run/docker.sock"
```

Comment:
This host may be different for differnt people. This is another thing I've had discussions about parameterizing.